### ISSUE TITLE
Custom Schematics on Servers Are Not Being Sent to Clients

### ISSUE THREAD

**Issue by TheRooster13** [ldtteam/Structurize#471]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.18.1
- MineColonies Version: minecolonies-1.18.1-1.0.328-ALPHA
- Structurize Version: structurize-1.18.1-1.0.319-BETA
- Related Mods and their Versions:

## Expected behavior
I would see my custom blueprint as a preview on the client.

## Actual behavior
The default schematic is shown as a preview on the client. However, when actually built, it uses the custom schematic.

## Steps to reproduce the issue
1. Extract a schematic for a building from the jar file.
2. Place the schematic in a creative single player world using the "Keeping Placeholder Blocks" option.
3. Modify something visible, scan, and save the new custom schematic (using the original name of the building).
4. Verify the schematic was changed properly by building it in a new location. The new schematic will be in the "My Schematics" section in the UI.
5. Copy the custom schematic from the "new" folder to the server into the correct folder using ftp or similar.
6. Connect to the server in-game.
7. Use the build tool to preview the building you modified. It should now be in the standard location in the UI.

Optional: Instantly build the building using creative mode to see what version gets built. It will build your custom building correctly. It's just the preview that is wrong.

### Logs

N/A

### Notes
The client's structurize/ServerUUID/schematics/ folders have no blueprint files in it. So I am guessing that are just not being sent to the clients.

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Raycoms** [comment]
actual switching back and forth shows it. But yeah, it gets sent over, but the preview is not updated once it's available.

**Comment by TheRooster13** [comment]
Switching back and forth how? I'm not seeing the preview update when I change the building type, or style, or level.

**Comment by TheRooster13** [comment]
Are these setting from structurize-server.toml messing it up?
	ignoreSchematicsFromJar = false
	allowPlayerSchematics = true

**Comment by Raycoms** [comment]
Basically: Go to your custom level 2, go back to level 1 and it should be loaded.

**Comment by TheRooster13** [comment]
Doesn't change. Still shows the default from the jar file.

Should there be files in the /structurize/ServerUUID/schematics folder? Or the /structurize/ServerUUID/cache folder? They're empty on my client.

**Comment by Raycoms** [comment]
Oh wait, you're trying to override schematics with schematics on your client?

**Comment by TheRooster13** [comment]
yes, I just added a note above.

**Comment by TheRooster13** [comment]
No. I am overriding schematics from the jar file with custom schematics on the server with the same name.

**Comment by Raycoms** [comment]
okay, so this is server -> client synch, and not client -> server sync, understood

**Comment by TheRooster13** [comment]
It seems like it, yes. Because the building is actually built as defined by the custom schematic. It just doesn't preview correctly. No matter how many times I swap options around. Even closing (red X) and restarting the build tool doesn't update the preview. Restarting the server and/or client doesn't help either.
